src/ucp/rma/flush.c

```diff
@@ -384,10 +384,6 @@ static ucs_status_t ucp_worker_flush_check(ucp_worker_h worker)
     ucp_worker_iface_t *wiface;
     ucs_status_t status;
 
-    if (worker->flush_ops_count) {
```

Comment:
why is removed here and moved outside?

Reply:
we want to check for flush_ops count separately from `uct_iface_flush()` that we check in this function

Reply:
why, seems can just leave it returning UCS_INPROGRESS

Reply:
> why, seems can just leave it returning UCS_INPROGRESS?

added a comment to make it clear
if it returns UCS_INPROGESS, but going over all EPs were done, we complete with `UCS_OK`, but we shouldn't